specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,190 @@
+# ApplicationLanguages
+
+This new type (`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages`) is a wrapper for
+the OS `Windows.Globalization.ApplicationLanguages` type with the updated behavior for the
+`PrimaryLanguageOverride` property to be supported for un-packaged apps.
```

Comment:
Where WinAppSDK has a replacement for a Platform SDK API, we should match the namespace other than prefixing with `Microsoft.`. Moving this from Globalization to Resources sounds like an org alignment

Reply:
RECOMMEND: Update to Microsoft.Windows.Globalization.ApplicationLanguages.